//header

#ifndef HEKATEROS_CONTROL_H_
#define HEKATEROS_CONTROL_H_

#include "Hekateros/hekateros.h"
#include "Hekateros/hekXmlCfg.h"
#include "Hekateros/hekRobot.h"

#include "rnr/rnrconfig.h"
#include "rnr/log.h"

using namespace ::hekateros;
HekRobot *pRobot = NULL; // one (application-global) robot to rule them all

#endif //  HEKATEROS_CONTROL_H_
